zkevm-circuits/src/pi_circuit.rs

```diff
@@ -80,12 +80,14 @@ pub struct PublicData {
 impl PublicData {
     /// Returns struct with values for the block table
     pub fn get_block_table_values(&self) -> BlockValues {
-        let mut history_hashes: Vec<H256> = self
-            .history_hashes
-            .iter()
-            .map(|&hash| H256::from(hash.to_be_bytes()))
-            .collect();
-        history_hashes.extend(vec![H256::zero(); 256 - history_hashes.len()]);
+        let history_hashes = [
```

Comment:
I have seen this in the [spec](https://github.com/privacy-scaling-explorations/zkevm-specs/blob/947e012c6d8d1566b295f8bfd717a82eeb475e18/src/zkevm_specs/evm/typing.py#L116) and also in this repo but don't know the exact locations.

The comment `It contains most recent 256 block hashes in history, where the lastest one is at history_hashes[-1].` which popups in multiple places also indicates (for me) that the **latest** block hash should be the last element. Unless it's not a typo and instead meant to be the oldest one? 

It confused me too in the beginning because I also saw the opposite happening, would be nice to get clarification on this. cc @ed255

Reply:
But I also think that having the latest one at `0` is more intuitive. Just a comment 🙂

Reply:
I now think the latest one at `0` is more intuitive. However, there are a few logic depending on the current order (circuit input builder, geth utils, etc...), so I'd prefer to change the order in another specific PR.